modules/ROOT/pages/basic-alerts.adoc

```diff
@@ -0,0 +1,42 @@
+[[basic-alerts]]
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+= Set up Basic Alerts for Servers and Mule Apps
+
+You can set up basic alerts to trigger email notifications when a metric you are measuring passes a specified threshold.
+
+You can create basic alerts for the following metrics for servers or Mule apps:
+
+* For on-premises servers and CloudHub apps:
+** CPU utilization
+** Memory utilization
+** Thread count
+* For on-premises, Runtime Fabric, and CloudHub apps:
+** Message count
+** Message error count
+** Message response time
+
+== Create a Basic Alert
+
+When you create a basic alert, you can select a source for the basic alert, configure the condition under which the basic alert is triggered, and specify which email addresses receive notifications for basic alert triggers.
+
+To create a basic alert:
+
+. Navigate to Anypoint Monitoring.
+. Click the *Alerts* tab.
+. Click the *+ New Alerts* button.
```

Comment:
Oh! I edited out the + in a button name earlier....is + part of the label???

Reply:
Yes.